### ISSUE TITLE
ecommerce.py:33-34: Rename position_key to 'option' at js...

### ISSUE THREAD

**Issue by 0pdd** [fidals/stroyprombeton#526]
The puzzle `468-ef699901` from #468 has to be resolved: 

https://github.com/fidals/stroyprombeton/blob/1322be8af4c4722911367f4578051c63d8c02139/stroyprombeton/views/ecommerce.py#L33-L34

The puzzle was created by duker33 on 21-Mar-19. 

  

If you have any technical questions, don't ask me, submit new tickets instead. The task will be \"done\" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

**Comment by 0pdd** [comment]
@artemiyds2 @duker33 please pay attention to this new issue.

**Comment by duker33** [comment]
blocked by #616 
We can't do renaming with no tests
